src/iqm/qiskit_iqm/iqm_provider.py

```diff
@@ -284,13 +305,13 @@ class IQMFacadeBackend(IQMBackend):
 
     def __init__(self, client: IQMClient, **kwargs):
         self.fake_adonis = IQMFakeAdonis()
-        target_architecture = client.get_quantum_architecture()
+        target_architecture = client.get_dynamic_quantum_architecture(kwargs.get('calibration_set_id', None))
 
         if not self.fake_adonis.validate_compatible_architecture(target_architecture):
             raise ValueError('Quantum architecture of the remote quantum computer does not match Adonis.')
 
         super().__init__(client, **kwargs)
-        self.name = f'IQMFacade{target_architecture.name}Backend'
+        self.name = 'IQMFacadeBackend'
```

Comment:
DQA does not contain a name?
I've made a bug report that the facade only works for Adonis, because there is not reason for that.

Reply:
Yes, DQA does not contain a name, only a calset id.

Reply:
Changed since only Adonis is supported for facade backend.